### PR TITLE
samples: led

### PR DESCRIPTION
**Pull request by wilu-nordic**

### CODE REVIEW

File: samples/led/README.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.. _npmx_led_sample:
+
+npmx: LED
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx LED sample demonstrates how to control PMIC LEDs using npmx drivers.
```
Comment (masz-nordic) [inline samples/led/README.rst:10]:
```suggestion
The LED sample demonstrates how to control LEDs using npmx drivers.
```

File: samples/led/src/main.c [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <npmx_driver.h>
+#include <npmx_led.h>
+
+#define LOG_MODULE_NAME pmic_led
```
Comment (masz-nordic) [inline samples/led/src/main.c:12]:
```suggestion
#define LOG_MODULE_NAME led
```

File: samples/led/src/main.c [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+		LOG_INF("PMIC device is not ready.");
+		return;
+	}
+
+	LOG_INF("PMIC device OK.");
+
+	/* Get the pointer to npmx device. */
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	/* Leds sequence to set. Only first 3 bits are used. */
```
Comment (annakielar) [inline samples/led/src/main.c:42]:
```suggestion
	/* Set the LEDs sequence. Only first 3 bits are used. */
```
To be confirmed if ok.

File: samples/led/src/main.c [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <npmx_driver.h>
+#include <npmx_led.h>
+
+#define LOG_MODULE_NAME pmic_led
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+/**
+ * @brief Set LEDs status based on lower 3 bits of status variable.
```
Comment (annakielar) [inline samples/led/src/main.c:16]: Do we want to stick to the header files' standards? Like: function for setting...

**Review by annakielar — CHANGES_REQUESTED** [review]

File: samples/led/README.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+|test_sample|
+
+#. |connect_kit|
+#. |connect_terminal|
+#. See blinking LEDs on the board.
```
[the remark this answers is not in the document]
Comment (wilu-nordic) [reply samples/led/README.rst:85]: It is just a result.

File: samples/led/src/main.c (hunk 4 above)
Comment (wilu-nordic) [reply samples/led/src/main.c:16]: @masz-nordic what do you think?

For me, we can leave it as it is.

File: samples/led/README.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
+.. _led_sample:
+
+LED
+###
+
+.. contents::
+   :local:
+   :depth: 2
+
+The LED sample demonstrates how to control LEDs using npmx drivers.
```
Comment (annakielar) [inline samples/led/README.rst:10]:
```suggestion
The LED sample demonstrates how to control LEDs with the use of npmx drivers.
```

File: samples/led/README.rst [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+|test_sample|
+
+#. |connect_kit|
+#. |connect_terminal|
+#. See blinking LEDs on the board.
```
Comment (annakielar) [inline samples/led/README.rst:85]:
```suggestion
#. |connect_kit|
#. |connect_terminal|

As a result, you should see LEDs blinking on the board.
```

File: samples/led/README.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (annakielar) [reply samples/led/README.rst:85]: If so, I would not treat it as a step to be done. Please check my suggestion below.

**Review by annakielar — CHANGES_REQUESTED** [review]

**Review by annakielar — APPROVED** [review]

**Pull request merged**